README.md

```diff
@@ -0,0 +1,65 @@
+# action-setup-zephyr (experimental)
+
+This actions initializes a Zephyr based project, downloading the Zephyr SDK and
+the necessary modules for a West based [Zephyr workspace application][1].
+
+# Basic usage
```

Comment:
indent all headings from that point by 1 level?

Reply:
Like `## Basic usage` and then `##` for all the others? I copied the title levels from https://raw.githubusercontent.com/actions/checkout/main/README.md

Reply:
> Like `## Basic usage` and then one more `#` for all the others? 

Ya, the title of the readme is the `#` entry, and you go down from there... at least that's I've always seen it done

> I copied the title levels from https://raw.githubusercontent.com/actions/checkout/main/README.md

I'd say this was bad luck then :) https://raw.githubusercontent.com/actions/deploy-pages/main/README.md https://raw.githubusercontent.com/github/codeql/main/README.md

Reply:
ha! ok I added a bunch of `#`